dashboard/config/blocks/GeneratedDancers/GeneratedDancers_alternateMoves.json

```diff
@@ -0,0 +1,142 @@
+{
```

Comment:
This is a modified version of the block from the main `Dancelab` pool. It uses the new image dropdown for the sprite group field, but it otherwise unremarkable.